app/routes/index.js

```diff
@@ -16,30 +16,81 @@
 
 const express = require('express');
 const router = express.Router();
+const asyncHandler = require('express-async-handler');
 
 const getBunyanConfig = require('../utils/bunyan.js').getBunyanConfig;
-const bunyan = require('bunyan');
-const logger = bunyan.createLogger(getBunyanConfig('/'));
+const ebl = require('express-bunyan-logger');
+
+const MongoClientClass = require('../mongo/mongoClient.js');
+const mongoConf = require('../conf.js').conf;
+const MongoClient = new MongoClientClass(mongoConf);
+
+const getOrg = require ('../utils/orgs').getOrg;
 
 const Kube = require('./kube/kube.js');
 const Install = require('./install');
 const Cron = require('./cron/cron.js');
-const Status = require('./v2/status.js');
 const Clusters = require('./v2/clusters.js');
 
-router.use('/kube', Kube);
-router.use('/install', Install);
+router.use('/api/kube', Kube);
+router.use(ebl(getBunyanConfig('/api/v2/')));
+
+router.use(asyncHandler(async (req, res, next) => {
+  const db = req.app.get('db');
+  req.db = db;
+  next();
+}));
 
 router.use((req, res, next) => {
```

Comment:
are we intending to make `razee-org-key` required for the the `/api/kube` route ?

Reply:
No. /api/kube will be the only route that does not require an orgKey